agent/consul/state/deep-copy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+readonly PACKAGE_DIR="$(dirname "${BASH_SOURCE[0]}")"
+cd $PACKAGE_DIR
+
+# Uses: https://github.com/globusdigital/deep-copy
+deep-copy \
+  -pointer-receiver \
+  -o ./catalog_schema.deepcopy.go \
```

Comment:
nit: Is there a particular reason for this to be `catalog_schema`?
```suggestion
  -o ./catalog.deepcopy.go \
```

Reply:
Actually, ignore me! Just spotted the type is defined in `catalog_schema.go` 🙈